pkgs/applications/backup/timeshift/wrapper.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenvNoCC
+, lib
+, wrapGAppsHook
+, timeshift-unwrapped
+, gdk-pixbuf
+, librsvg
+, shared-mime-info
+}:
+propagatedBuildInputs:
+
+stdenvNoCC.mkDerivation {
+  inherit (timeshift-unwrapped) pname version;
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+  ];
+
+  inherit propagatedBuildInputs;
+
+  ## Wrappable `lndir "${timeshift-wrapped}" "$out"`
+  ## with "$out/nix-support" removed
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    (
```

Comment:
We don't cd here. Why do we need this?

Reply:
Maybe I used to do `cd` here before. I'll remove that.